docs/extensions/registration.md

```diff
@@ -0,0 +1,351 @@
+---
+title: Registering Extension Controllers
+---
+
+# Registering Extension Controllers
+
+Before Gardener can manage shoot clusters, it needs to know which required and optional extensions are available in the landscape.
+The following sections explain the general registration process for extensions.
+
+## `Extension`s
+
+The registration starts by creating [`Extension`](../../example/operator/15-extension.yaml) resources in the garden runtime cluster. 
+They represent the single source for all deployment aspects an extension may offer (garden, seed, shoot, admission).
+The `gardener-operator` takes these resources to deploy the extension to the garden runtime cluster, as well as creating corresponding [`ControllerRegistration`](../../example/25-controllerregistration.yaml) and [`ControllerDeployment`](../../example/25-controllerdeployment.yaml) resources in the virtual garden cluster.
+
+Please see the following example of an `Extension` resource which mainly configures:
+- The resource kinds and types the extension is responsible for
+- A Reference the OCI Helm chart for the extension
+- Values for the extension in the garden runtime cluster
+- Values for the deployment in the seed clusters
+- A Reference the OCI Helm chart(s) for the extension admission
+
+```yaml
```

Comment:
Can you please move values fields below chart related fields as they seem to be optional? Also after that can you align the order of the bullets that explain the example to match the example field order?